src/metrics/promotions.ts

```diff
@@ -104,9 +138,9 @@ export class PromotionMetric implements Metric {
     return {
       promotedToTotals: this.totals,
       uasPromotingPieces: this.promotionMap,
-      maxNumQueens: this.maxQueenCounts,
-      movesAndGamesWithMaxQueenCount: this.movesAndGamesWithMaxQueenCount,
-    }
+      maxNumQueens: this.maxQueensOnBoard,
+      movesAndGamesWithMaxQueenCount: this.movesAndGamesWithMaxQueensOnBoard,
+    };
```

Comment:
let's return movesAndGamesWithMaxQueens and this.maxQueens too?

Reply:
if we aren't using them doesn't really matter